<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ClientIndicator[state = '0']{
	background-color: rgb(255, 0, 0);
}

#ClientIndicator[state = '1']{
	background-color: rgb(0, 85, 0);
}

#ServerIndicator[state = '0']{
	background-color: rgb(255, 0, 0);
}

#ServerIndicator[state = '1']{
	background-color: rgb(0, 85, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Info box</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Client/Server setup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="IpAddressLabel">
              <property name="text">
               <string>Ip Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ClientIpAddress">
              <property name="inputMask">
               <string>000.000.000.000;_</string>
              </property>
              <property name="state" stdset="0">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="ClientIndicator">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="state" stdset="0">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="ClientPortLabel">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="ClientPort">
              <property name="minimum">
               <number>1024</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>5051</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="BtnConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="ClientSendData_GroupBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Send data to connected device</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="ClientTextLabel">
                 <property name="text">
                  <string>Text</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ClientTextToSend">
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="state" stdset="0">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="BtnClientSend">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="ServerSendData_GroupBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>Send data to clients</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="ServerTextLabel">
                 <property name="text">
                  <string>Text</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ServerText"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="BtnServerSend">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Server status info</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Server status</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QPushButton" name="BtnUpdateServInfo">
                   <property name="text">
                    <string>Update server info</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPlainTextEdit" name="ServerRunInfo">
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>200</height>
                    </size>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                   <property name="placeholderText">
                    <string>After server run here you can find ip and port of hosted session</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="ServerPortLabel">
                     <property name="text">
                      <string>Port</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="ServerPort">
                     <property name="minimum">
                      <number>1024</number>
                     </property>
                     <property name="maximum">
                      <number>65535</number>
                     </property>
                     <property name="value">
                      <number>5051</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="BtnServerRun">
                     <property name="text">
                      <string>Run server</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="ServerIndicator">
                     <property name="minimumSize">
                      <size>
                       <width>20</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>20</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="state" stdset="0">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QListWidget" name="LstConsole"/>
                </item>
                <item>
                 <widget class="QPushButton" name="BtnClearConsole">
                  <property name="text">
                   <string>Clear</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection_window">
    <property name="title">
     <string>Connection window</string>
    </property>
    <addaction name="actionCheck_player_info_tab"/>
   </widget>
   <addaction name="menuConnection_window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCheck_player_info_tab">
   <property name="text">
    <string>Check player info tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
